v2/car_test.go

```diff
@@ -1,6 +1,7 @@
 package car_test
 
 import (
+	"bytes"
 	cbor "github.com/ipfs/go-ipld-cbor"
 	car_v1 "github.com/ipld/go-car"
 	car_v2 "github.com/ipld/go-car/v2"
```

Comment:
nit: you should avoid foo_bar names in Go, even for packages; I think carv1 and carv2 would work well here. I'd only resort to foo_bar in very extreme cases, where there's no "single word" one can use, but hopefully that should never happen.